### PR TITLE
Enable changing language in the settings

### PR DESCRIPTION
**Pull request by jrainville**
Enables switching to any of the supported languages.
Right now, the first language is the system one, then you can change to the one you want.
A known issue is that the select will default to `en` even though your system could be something else, but it was not worth for now to add another C++ method to get the current locale.

 An improvement would be to change the locales array to an array of object where there is the locale  + the formatted name, because right now it's just `fr`, `es`, etc.

Branched off of: https://github.com/status-im/nim-status-client/pull/529

image

### CODE REVIEW

**Comment by jrainville** [comment]
BTW, the first commit is just re-running the script, so you can ignore it

File: src/nim_status_client.nim [hunk 1]
```diff
@@ -43,7 +43,10 @@ proc mainProc() =
   var node = node.newController(status)
   engine.setRootContextProperty("nodeModel", node.variant)
 
-  var profile = profile.newController(status)
+  proc changeLanguage(locale: string) =
+    engine.setTranslationPackage(fmt"/home/jonathan/dev/nim-status-client/ui/i18n/qml_{locale}.qm")
```
Comment (richard-ramos) [inline src/nim_status_client.nim:47]: ???

Comment (jrainville) [reply src/nim_status_client.nim:47]: ONLY MY PC WORKS! Muhahaha

Comment (jrainville) [reply src/nim_status_client.nim:47]: But seriously, yeah I was tired when I finished the PR so I forgot to change it back to relative haha

Comment (richard-ramos) [reply src/nim_status_client.nim:47]: > ONLY MY PC WORKS! Muhahaha

We'll ship your PC :smiling_imp:

Comment (jrainville) [reply src/nim_status_client.nim]: Fixed

**Pull request merged**